### ISSUE TITLE
Sync with Cloud Icon

### ISSUE THREAD

**Issue by theDevelopper** [lucide-icons/lucide#2055]
### Icon name

cloud-sync

### Use cases

Similar to the `folder-sync` icon the cloud sync would indicate a (two-way) sync with the cloud. 
At the moment there are two cloud icons that either represent an upload or a download but no icon for both.

The use-case is that often one wants to sync information both ways (up and down) at the same time. This could be represented in two ways: either with an up and down arrow next to each other or -- like with the `folder-sync` -- a cloud and two arrows in a circle.

Why do other icons not convey the same message?
The `folder-sync` strongly suggests the syncing of a folder which is not always the case when syncing with the cloud. The simple reload icon might indicate updating something but does not indicate that the cloud is involved strongly enough.

### Design ideas

Like with the `folder-sync` but a cloud and two arrows in a circle, centered like with `cloud-upload`/`cloud-download`

### Checklist

- [X] I have searched if someone has submitted a similar issue before and there weren't any. (Please make sure to also search closed issues, as this issue might already have been resolved.)
- [X] I have searched existing icons to make sure it does not already exist and I didn't find any.
- [X] I am not requesting a brand logo and the art is not protected by copyright.
- [X] I am not requesting an icon that includes religious, political imagery or hate symbols.
- [X] I have provided appropriate use cases for the icon(s) requested.

**Comment by ericfennis** [comment]
`cloud-backup` and `cloud-sync` based on `database-backup` and `folder-sync`
<img width="203" alt="image" src="https://github.com/lucide-icons/lucide/assets/11825403/48b84f2a-75f3-4c8a-8744-63eb6557f785">

100%
![cloud icons](https://github.com/lucide-icons/lucide/assets/11825403/6e183abd-cdd3-437b-9c12-c6b1abacbf6b)

**Comment by karsa-mistmere** [comment]
> `cloud-backup` and `cloud-sync` based on `database-backup` and `folder-sync` <img alt="image" width="203" src="https://private-user-images.githubusercontent.com/11825403/321941268-48b84f2a-75f3-4c8a-8744-63eb6557f785.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI5MTU5MTMsIm5iZiI6MTcxMjkxNTYxMywicGF0aCI6Ii8xMTgyNTQwMy8zMjE5NDEyNjgtNDhiODRmMmEtNzVmMy00YzhhLTg3NDQtNjNlYjY1NTdmNzg1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEyVDA5NTMzM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRmMDNiNjIwNWUzZTIzYzAwMWNjNjI0M2MyNjk5M2UyMGQzMmVmNTBkM2RjZTcyMmQyYjRkYmFhOGU4Yjc4NDQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.7zXRGLS9lJ38lXN-byPC3Sz0zE63hhUc0oGxo0nKUqY">
> 
> 100% ![cloud icons](https://private-user-images.githubusercontent.com/11825403/321942386-6e183abd-cdd3-437b-9c12-c6b1abacbf6b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI5MTU5MTMsIm5iZiI6MTcxMjkxNTYxMywicGF0aCI6Ii8xMTgyNTQwMy8zMjE5NDIzODYtNmUxODNhYmQtY2RkMy00MzdiLTljMTItYzZiMWFiYWNiZjZiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEyVDA5NTMzM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVmMTkwMWMzNzNiYzhiNzgyY2Y4ZDkxNzQxMTA4MGM1ZmE2YmVmNTE4ZmIzOTljY2JkYWZmMzEzZWY2ZmVjMDQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.PwYpG4WrLO87mF99wVremYTMZYl5PCB93JvT8cQa1BQ)

Can we maybe try one that's not centered so that more of the cloud shape remains for better recognisability?

**Comment by karsa-mistmere** [comment]
> @karsa-mistmere I've based this on the other cloud icons, which are also centered. <img alt="image" width="321" src="https://private-user-images.githubusercontent.com/11825403/321980902-9e3b238e-c217-45ad-bf14-0b06ddc01167.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI5MjUxNTgsIm5iZiI6MTcxMjkyNDg1OCwicGF0aCI6Ii8xMTgyNTQwMy8zMjE5ODA5MDItOWUzYjIzOGUtYzIxNy00NWFkLWJmMTQtMGIwNmRkYzAxMTY3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEyVDEyMjczOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI3ZmI1MTYwZDNhZjc3NDRlMDc5YTJjZDk4MzM5MmJhYjIzMGE4OTg1Yzc4MzQyZGYxY2Q0ODJmZjMwY2UzOWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.V-cvRy9t-fT5iBmdn9s4Ofj5XERhyE0nbbufoHCfLE4">

Yeah, I know, it's just that the arrows are much larger here and I feel like the cloud shape could be somewhat more apparent.

**Comment by theDevelopper** [comment]
I like the arrows in the middle. Keeps them in line with the other cloud icons.
I think if you move them to the right you actually lose the cloud shape. Unless the cloud is flipped or the arrows are on the left. But that would create even more inconsistencies.

**Comment by theDevelopper** [comment]
@ericfennis any updates on this?

**Comment by jguddas** [comment]
> @ericfennis any updates on this?

You can make a PR if you want to, use case feels solid.

**Comment by theDevelopper** [comment]
I am sorry. I am not a designer and I cannot "draw". Neither can I create SVG's. Thats why I requested the addition of these icons. I therefore cannot create a PR. I thought that this was the purpose of an Icon Request?

**Comment by pemperor-alpetine** [comment]
@jguddas stumbled upon this here, this might be useful

![Image](https://github.com/user-attachments/assets/24a5967b-632b-424b-8d4d-6bab5a9dab86)

**Comment by ericfennis** [comment]
I will try make a PR!
